conda_smithy/templates/run_osx_build.sh.tmpl

```diff
@@ -2,38 +2,58 @@
 
 set -x
 
-{{ fast_finish }}
```

Comment:
We don't need this anymore here. Travsi and azure never used it or don't now. Circle has its own step before this one.